src/demos/calcite-checkbox.html

```diff
@@ -30,6 +30,14 @@
   <script nomodule src="../build/calcite.js"></script>
   <script type="module" src="./_assets/demo-form.js"></script>
   <script type="module" src="./_assets/demo-spacer.js"></script>
+  <style>
+    calcite-checkbox input[type="checkbox"] {
+      opacity: 1 !important;
+      margin: 50px !important;
+      padding: 50px !important;
+      position: static !important;
```

Comment:
I think it can still be displayed if they set the zIndex, right?

Could also do some things like ..

```
transform: scale(0);
visibility:hidden;
```

Does the input need to be added to the element at all?
Could it be in the shadow DOM?

Do we need to add something like`aria-hidden` or `role="none"` to remove any screen reader stuff?

Reply:
Thanks for catching the z-index issue, updated that so it should now prevent setting that from outside also.  Also added `transform`.  Can't set `visibility: hidden` because that breaks keyboard focus.

>Does the input need to be added to the element at all?
It is rendered outside the element's shadowRoot, which effectively makes it a slotted child, so it is already part of the element.

>Could it be in the shadow DOM?
Defeats the purpose of allowing it to be picked up by parent `<form>`s.

>Do we need to add something like `aria-hidden` or `role="none"` to remove any screen reader stuff?
Since this input is technically active in the DOM, meaning the keyboard can focus it, I think it is best to leave it as-is so that a screen reader can do its thing properly, which is probably pretty similar to what the keyboard does when it focuses them.